runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -593,7 +594,7 @@ public void stop() {
 
         // Write this message out on the channel
         channel.writeAndFlush(request, channel.voidPromise());
-        log.trace("Sent request message: {}", request.getHeader());
+        log.trace("Sent request message: {}", TextFormat.shortDebugString(request.getHeader()));
```

Comment:
This will be computed even if it wont be logged. I think its better shorten the string if trace is enabled.

Reply:
Added, thanks!